### DISCUSSION TITLE
Incognito vs guest vs anonymous

### DISCUSSION THREAD

**Discussion by Memo-Es** [opencollective/opencollective#4817] [General]
I wanted to document a discussion I want to raise on the incognito feature we have on the platform to unify visual and textual language.

When someone contributes, they see this option:

<img width="430" alt="Screen Shot on 2021-10-12 at 10-49-44" src="https://user-images.githubusercontent.com/49539954/136990161-38f567fa-1b94-4694-b4f8-64e3ccae6cfb.png">

And when that contribution is surfaced in the transaction list, it appears like this:

<img width="439" alt="Screen Shot on 2021-10-12 at 10-54-46" src="https://user-images.githubusercontent.com/49539954/136990275-f12e9bc7-fd0f-42ae-9177-102d40569279.png">

I think we also use 'anonymous' or 'anonymously' somewhere, but I'm not sure about that one.

Anyway, my point is to unify this concept and ideally have the wording and imagery consistent.

@BenJam @alanna @Betree

**Comment by alanna** [comment]
Incognito and guest are different, as far as I understand. But maybe I am wrong about it? Let me know if this is correct:

* Incognito: Registered user, we record their identity and the transaction is associated to their profile, but publicly it shows as incognito and people an't tell that profile is associated with the transaction. You must be logged in to use incognito. If logged in, the contributor can see their incognito transactions.
* Guest: The user is not registered or logged in. We collect the minimum information (ranging from none to more if it's a higher amount). The user can't log in to manage their profile and their various guest contributions are not associated with each other at all. If they later create an account, the system uses their email to connect them to their previous guest transactions and they will not be private/anonymous. People shouldn't use guest when they want incognito for this reason.

**Comment by Betree** [comment]
@alanna is correct. @Memo-Es in the example you shared, picking "Incognito" should display your contribution as "Incognito", not "Guest". If that's not the case then it's a bug.

Regarding "anonymous", it used to be the default name in the old contribution flow for users who did not set a name. It's not in use anymore and I don't think we're surfacing that somewhere, except maybe for a few users that still have that set in their name field.

**Comment by alanna** [comment]
I'm pretty sure a lot of users would still be confused about the difference
between incognito and guest as well, so if there are ways we can make it
clearer we should try to do that.
